src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -328,9 +328,7 @@ const UserLayout: FC = ({ children }) => {
         {
           title: t('me:dataExplorer'),
           path: '/profile/treemapper/data-explorer',
-          hideItem: !(
-            process.env.ENABLE_ANALYTICS === 'true' && user?.type === 'tpo'
-          ),
+          hideItem: !(process.env.ENABLE_ANALYTICS && user?.type === 'tpo'),
         },
```

Comment:
I feel we should explicitly check ENABLE_ANALYTICS for a specific value and otherwise disable it. In this implementation, any value will enable analytics.

Reply:
Not applicable as ENABLE_ANALYTICS is a boolean value coming from next.config.js